### PR TITLE
Indices20a24

### PR DESCRIPTION
**Pull request by sbiner**

### CODE REVIEW

File: xclim/utils.py [hunk 1]
```diff
@@ -127,6 +131,191 @@ def daily_downsampler(da, freq='YS'):
 class UnivariateIndicator(object):
     r"""xclim indicator class"""
 
+
+def get_ev_length(ev, verbose=True, method=2, timing=True, using_dask_loop=True):
```
[the remark this answers is not in the document]
Comment (sbiner) [reply xclim/utils.py]: Je ne connaissais pas run_length quand j'ai fait ça. On peut l'ajouter comme méthode et voir ce que ça donne ... ou simplement utiliser run_length.

File: xclim/indices.py [hunk 2]
```diff
@@ -334,6 +342,52 @@ def growing_season_length(tas, thresh=5.0, window=6, freq='YS'):
...
+
+    Parameters
+    ----------
+
+    tasmin : xarrray.DataArray
+      Minimum daily temperature [℃] or [K[
+    tasmax : xarrray.DataArray
+      Maximum daily temperature [℃] or [K[
+    thresh_tasmin : float
+      Threshold temperature for tasmin on which to designate a heatwave [℃] or [K[
```
[the remark this answers is not in the document]
Comment (Zeitsperre) [reply xclim/indices.py]: I think I may have written that. While I may seem laconic in person, my technical writing is often needlessly verbose. The best of both worlds!

```suggestion
      The minimum temperature threshold needed to trigger a heatwave event [℃] or [K]
```

File: xclim/indices.py [hunk 3]
```diff
@@ -334,6 +342,52 @@ def growing_season_length(tas, thresh=5.0, window=6, freq='YS'):
...
+      Threshold temperature for tasmax on which to designate a heatwave [℃] or [K[
+    window : int
+      Minimum number of days with temperatures above thresholds to qualify as a heatwave.
+    freq : str, optional
+      Resampling frequency
+
+    Returns
+    -------
+    xarray.DataArray
+      Number of heatwave at the wanted frequency
```
[the remark this answers is not in the document]
Comment (tlogan2000) [reply xclim/indices.py]: I suggest something like : 'and determine the <operation> at the provided frequency' 
e.g. :  max_n_day_precipitation_amount - "Calculate the n-day rolling sum of the input daily total precipitation series .. and determine the maximum at the provided frequency"

We would almost need this in every doc string.  I think pretty much everything does a resample correct?

File: xclim/indices.py [hunk 4]
```diff
@@ -334,6 +342,52 @@ def growing_season_length(tas, thresh=5.0, window=6, freq='YS'):
     return d.resample(time=freq).max(dim='time')
 
 
+def heat_wave_frequency(tasmin, tasmax, thresh_tasmin=22.0, thresh_tasmax=30,
+                        window=3, freq='YS', use_rl=True, **kwds):
+    # Dev note : we should decide if it is deg K or C
+    r"""Heat wave frequency
+
+    Number of heat wave over a given period. A heat wave is defined as an event
```
[the remark this answers is not in the document]
Comment (huard) [reply xclim/indices.py]: I'm confused by this function. Is there a reference implementation somewhere ? It seems weird to me that the function would return 1 no matter if the heatwave lasts 3 days or 21 days.

[the remark this answers is not in the document]
Comment (huard) [reply xclim/indices.py]: If we have no references, this would be one to look at https://journals.ametsoc.org/doi/10.1175/JCLI-D-12-00383.1

**Pull request merged**